src/Svg/Surface/CPdf.php

```diff
@@ -590,13 +590,15 @@ protected function o_viewerPreferences($id, $action, $options = '')
     /**
      * define the document catalog, the overall controller for the document
      *
-     * @param $id
-     * @param $action
+     * @param string $id
+     * @param string $action
      * @param string|array $options
      * @return string|null
      */
     protected function o_catalog($id, $action, $options = '')
     {
+        $o = [];
```

Comment:
Seems unnecessary. `$o` is retrieved when it's needed, which is when `$action != 'new'`. Otherwise, SvgLib is doing a set on an uninitialized multi-dimensional key which PHP handles automatically. Which has no impact on the generated PDF, but that's a logic flow issue (though validation could be implemented).

Still, I see no harm in the change.